source/guides/guides/continuous-integration.md

```diff
@@ -751,6 +751,15 @@ After all tests across all Cypress instances finish, kill the Xvfb background pr
 pkill Xvfb
 ```
 
+{% note warning %}
+In certain Linux environments, you may experience connection errors with your X11 server. In this case, you may need to start Xvfb with the following command:
```

Comment:
@CypressHarry I think it would be helpful to add a sentence or two giving users a bit of explanation on how this line can help remedy the issue.